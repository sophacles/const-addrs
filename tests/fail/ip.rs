use const_addrs::ip;

fn main() {
    let _ = ip!("192.168.11");
    let _ = ip!("2001:db8::32::23");
}
