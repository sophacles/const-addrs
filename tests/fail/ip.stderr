error: invalid IP address syntax
 --> tests/fail/ip.rs:4:17
  |
4 |     let _ = ip!("192.168.11");
  |                 ^^^^^^^^^^^^

error: invalid IP address syntax
 --> tests/fail/ip.rs:5:17
  |
5 |     let _ = ip!("2001:db8::32::23");
  |                 ^^^^^^^^^^^^^^^^^^
